Documentation/development-flow.md

```diff
@@ -293,7 +293,6 @@ Signed-off-by: First Name Last Name <email address>
 The `component` **MUST** be one of the following:
```

Comment:
This list is out of date with others that have recently been added. How about if we remove the list here and change to the following (and let's always just point to master since that's where we expect PRs to be merged first):
```suggestion
The `component` **MUST** be in the [list checked by the CI](https://github.com/rook/rook/blob/master/.commitlintrc.json):
```

Reply:
updated.